.drone.star

```diff
@@ -3,14 +3,38 @@ FULL = 1
 FEDERATED = 2
 NOTIFICATIONS = 3
 
+ALPINE_GIT = "alpine/git:latest"
+DEEPDRIVER_DOCKER_ORACLE_XE_11G = "deepdiver/docker-oracle-xe-11g:latest"
+DRONE_CLI_ALPINE = "drone/cli:alpine"
+HENRICH_DOCKER_SAUCE_CONNECT = "henrrich/docker-sauce-connect:latest"
+MELTWATER_DRONE_CACHE = "meltwater/drone-cache:v1"
+MINIO_MC_RELEASE_2020_VERSION = "minio/mc:RELEASE.2020-12-10T01-26-17Z"
+MINIO_MC_RELEASE_2021_VERSION = "minio/mc:RELEASE.2021-03-23T05-46-11Z"
```

Comment:
do we need both versions or can we use the same in all cases?

Reply:
I used the 2021 version in all cases and it working fine.